crates/iota-protocol-config/src/snapshots/iota_protocol_config__test__version_13.snap

```diff
@@ -34,6 +34,7 @@ feature_flags:
   normalize_ptb_arguments: true
   select_committee_from_eligible_validators: true
   track_non_committee_eligible_validators: true
+  consensus_median_based_commit_timestamp: true
```

Comment:
What about other snapshot files? `iota_protocol_config__test__Testnet_version_13.snap` and `iota_protocol_config__test__Mainnet_version_13.snap`